.github/workflows/dao-contracts.yml

```
@@ -0,0 +1,35 @@
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [18.x, 20.x, 22.x]
+
+    defaults:
+      run:
+        working-directory: dao-contracts
+
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)